src/steamship_langchain/file_loaders/unstructured.py

```diff
@@ -0,0 +1,54 @@
+"""File loader that uses the `unstructured` package to extract text content."""
+from typing import Any, Dict, List, Optional
+
+from pydantic import Field
+from steamship import Block, File, MimeTypes
+
+from steamship_langchain.file_loaders import BaseFileLoader, generate_file_tags
+
+
+class UnstructuredFileLoader(BaseFileLoader):
+    """File loader that uses `unstructured` to extract text content prior to upload.
+
+    NOTE: Requires install of unstructured package before use: `pip install unstructured`.
```

Comment:
Need a comment that for now this makes it undeployable as a Steamship package?

Reply:
I _thought_ about that, but think maybe it is OK to not call those out in the code, but rather in a follow-on docs PR that notes that the purpose of the importers is currently client-side. thoughts?